htdocs/install/doctemplates/websites/website_template-homesubmenu/containers/page202.tpl.php

```diff
@@ -128,7 +128,7 @@ class="dropdown-menu"
                         <p>
                             Lorem ipsum dolor, sit amet consectetur adipisicing
                             elit. Ab fuga nobis omnis alias, aliquid iste cumque
-                            tempora nam reprehenderit quia itaque debitis,
+                            tempora name reprehenderit quia itaque debitis,
```

Comment:
Possible to exclude this as this is latin and not english ?

Reply:
It's "Lorem Ipsum", not actually latin.

Codespell does not allow precise local exclusions: one needs to exclude the word or the file.

To me this was not important, and I prefer checking the rest of the file rather than excluding the file or the suggestion for something that could easily happen ("nam").

Another solution is to change "nam" to "nim".  Codepsell does not have a suggestion for that.

Reply:
I changed it to nim.  (+squash & rebase).